src/repositories/tags.repository.ts

```diff
@@ -0,0 +1,30 @@
+import { ObjectId } from "mongoose";
+
+import { Tag, TagModel } from "../models/Tag.model";
+
+export const getById = async (tagId: string | ObjectId) => {
+  return TagModel.findById(tagId);
+};
+
+export const create = async (tagData: Partial<Tag>) => {
```

Comment:
```suggestion
export const create = (tagData: Partial<Tag>) => {
```
not await - async  was not needed